README.md

```diff
@@ -2,92 +2,82 @@
 
 [![Build Status](https://travis-ci.org/AugurProject/augur-core.svg)](https://travis-ci.org/AugurProject/augur-core)
 
-Ethereum contracts for a decentralized prediction market platform.
+Smart contracts for [Augur](https://augur.net), a decentralized prediction market platform on the [Ethereum](https://ethereum.org) blockchain.
 
 ## Installation
 
-You should already have a system-wide installation of Python and it should be Python 2.7.6 or greater.
+You need a system-wide installation of Python (v2.7.6 or higher).  Install the dependencies:
```

Comment:
Since you seem to be on a Readme updating kick, probably worth mentioning here that you need [(Python and Node) or Docker] installed to work with this repository.  I believe we are currently all on Node 8, anything else may or may not work.

Reply:
My true passion is updating readmes